components/AddRoomModal/AddRoomModal.js

```diff
@@ -0,0 +1,126 @@
+import { useState } from "react";
+import styled from "styled-components";
+import { RoomType } from "../RoomTile/RoomTile";
+
+// Styled Components
+const OpenBtn = styled.button`
+  position: absolute;
+  top: 20px;
+  right: 50px;
+  padding: 5px 7px;
+  border: none;
+  background-color: lightgrey;
+`;
+
+const Modal = styled.div`
+  width: 100vw;
+  height: 100vh;
+  top: 0;
```

Comment:
As it is all 0, could it be left out? Or maybe it could me combined into one value?